.github/workflows/dependabot.yml

```
@@ -1,10 +1,13 @@
 name: Dependabot auto-merge
 on: pull_request
 
+permissions:
+  pull-requests: write
```

- https://github.com/maplibre/martin/pull/2280

```suggestion
  pull-requests: write
  contents: write
```